infra/tf-next/fly/ursa-gateway.tf

```diff
@@ -0,0 +1,26 @@
+resource "fly_machine" "ursa-gateway-machine" {
+  for_each = toset(var.regions)
+  app    = "ursa-gateway"
+  region = each.value
+  name   = "ursa-gateway-${each.value}"
+  image  = "ghcr.io/fleek-network/ursa-gateway:latest"
+  services = [
+    {
+      ports = [
+        {
+          port     = 443
+          handlers = ["tls", "http"]
+        },
+        {
+          port     = 80
+          handlers = ["http"]
+        }
+      ]
+      "protocol" : "tcp",
+      "internal_port" : 80
```

Comment:
We're not supporting 80 for the time being unless you want to redirect from 80 -> 443 by default.

Reply:
I will leave this in for the redirect